staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -303,6 +303,18 @@ type VolumeStatus struct {
 	HotplugVolume *HotplugVolumeStatus `json:"hotplugVolume,omitempty"`
 	// Represents the size of the volume
 	Size int64 `json:"size,omitempty"`
+	// If the volume is memorydump volume, this will contain the memorydump info.
+	MemoryDumpVolume *DomainMemoryDumpInfo `json:"memoryDumpVolume,omitempty"`
+}
+
+// DomainMemoryDumpInfo represents the memory dump information
+type DomainMemoryDumpInfo struct {
+	// DumpTimestamp is the time when the memory dump occured
+	DumpTimestamp *metav1.Time `json:"dumpTimestamp,omitempty"`
+	// ClaimName is the name of the pvc the memory was dumped to
+	ClaimName string `json:"claimName,omitempty"`
+	// TargetFileName is the name of the memory dump output
+	TargetFileName string `json:"targetFileName,omitempty"`
```

Comment:
I wonder if it makes sense to somehow also encode some/all of this information as annotations on the target pvc?  It doesn't appear that the pvc is annotated at all now.

Reply:
That is an option. If it's alright with you, i'll add it in a different PR to make it possible for this PR to be merged soon :)

Reply:
Yeah that sounds nice, I recently added labels to restore PVCs so we could have some queryable metrics about them
in https://github.com/kubevirt/kubevirt/pull/7533